theme.js

```diff
@@ -3,6 +3,9 @@ import styled from 'styled-components'
 export const Form = styled.form`
 	width: clamp(20rem, 20vw, 80vw);
 `
+export const Form3 = styled.div`
+	width: clamp(20rem, 20vw, 80vw);
+`
```

Comment:
K čemu jsou tu dvě stejné?

Reply:
Teda spíš proč je Form3 div

Reply:
Yes, to jsem zapomněl smazat.

Reply:
form3 byl div, protože jsem nemohl mít v sobě dva vnořené formy, když jsem pracoval s Formik